### PR TITLE
Many small updates.

### PR DESCRIPTION
**Pull request by deflaux**
Also removed several obsolete files.  The following redirects need to be configured before this change can be merged:
* pgp-data.html to use_cases/discover_public_data/pgp_public_data.html
* use_cases/browse_genomic_data/index.html to sections/access_data.html
* use_cases/run_familiar_tools/index.html to sections/access_data.html
* workshops/index.html to sections/learn_more.html

### CODE REVIEW

File: docs/source/includes/spark_setup.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-* Deploy your Spark cluster using `Google Cloud Dataproc`_.
+* Deploy your Spark cluster using `Google Cloud Dataproc`_.  This can be done using the `Cloud Console <https://console.developers.google.com/project/_/dataproc/clustersAdd>`__ or the following ``gcloud`` command:
```
Comment (mbookman) [inline docs/source/includes/spark_setup.rst]: Can you set the URL to console.cloud.google.com as per https://github.com/googlegenomics/readthedocs/issues/112?

**Comment by mbookman** [comment]
LGTM

File: docs/source/use_cases/compute_principal_coordinate_analysis/2-way-pca.rst [hunk 2]
```diff
@@ -22,10 +22,7 @@ __ RenderedVersion_
 
 .. contents::
 
-`Principal Coordinate Analysis <http://occamstypewriter.org/boboh/2012/01/17/pca_and_pcoa_explained/>`_
-counts the number of variants two samples have in common.  These counts are then placed into an
-``NxN`` matrix where ``N`` is the number of samples in the variant set.  The matrix is centered,
-scaled, and then the first two principal components are computed for each invididual.
+`Principal Coordinate Analysis <http://occamstypewriter.org/boboh/2012/01/17/pca_and_pcoa_explained/>`_ counts the number of variants two samples have in common.  These counts are then placed into an ``(N+M)x(N+M)`` matrix where ``N`` is the number of samples in the control variant set (e.g., 1,000 Genomes) and ``M`` is the number of samples in the case variant set.  The matrix is centered, scaled, and then the first two principal components are computed for each invididual.
```
Comment (pgrosu) [inline docs/source/use_cases/compute_principal_coordinate_analysis/2-way-pca.rst]: Nicole, when performing this Multidimensional scaling I'm not sure you're actually doing any scaling in the code - as referenced in the link below - besides the centering before running the PCA on the centered dissimilarity matrix:

https://github.com/googlegenomics/spark-examples/blob/15213450109e0d8629317f98c275be1cf5114072/src/main/scala/com/google/cloud/genomics/spark/examples/VariantsPca.scala#L221

Hope it helps,
~p

Comment (cmclean) [inline docs/source/use_cases/compute_principal_coordinate_analysis/2-way-pca.rst]: Nit: s/invididual/individual/

**Comment by cmclean** [comment]
Not a blocking comment, but I'm curious whether /includes/dataflow_on_gce_setup.rst could also be updated to suggest Java 8 rather than Java 7. The forthcoming LD pipeline uses Java 8 for writing to BigTable. Can the other pipelines be run on 8?

**Comment by deflaux** [comment]
Filed https://github.com/googlegenomics/readthedocs/issues/115 for Java 8.  

Saving https://github.com/googlegenomics/readthedocs/issues/112 for another PR.

I set up the redirects but they do not appear to be working at the moment.  (see also https://github.com/rtfd/readthedocs.org/issues/1826)

**Comment by pgrosu** [comment]
Nicole, Maybe an additional PR might be needed for the other points as well, so they don't get lost in the mists of time :)

~p

**Comment by deflaux** [comment]
@pgrosu  Thanks for pointing out the lack of scaling!  I filed https://github.com/googlegenomics/spark-examples/issues/82

**Comment by pgrosu** [comment]
Nicole, sure thing and sorry to be picky, but the above text says:

```The matrix is centered, scaled, and then the first two principal components are computed for each invididual.```

This suggests that you are scaling before you find that PC1 and PC2, which is confusing.  Usually you determine the eigenvectors with the 2 highest eigenvalues in this particular case, and then you scale the data by multiplying with them, not beforehand. Maybe my eyes are getting tired, but I'm not sure I see where you're actually performing that in the Dataflow code.  I notice that you find `maxEigenvalue` and `secondEigenvalue`, but I'm not sure where you rescale the original data with them.

Thanks,
~p

**Pull request merged**